module/applications/item/ability-use-dialog.mjs

```diff
@@ -186,7 +186,7 @@ export default class AbilityUseDialog extends Dialog {
     for ( const profile of profiles ) {
       const doc = profile.uuid ? fromUuidSync(profile.uuid) : null;
       if ( profile.uuid && !doc ) continue;
-      options[profile._id] = profile.name ?? doc?.name ?? "—";
+      options[profile._id] = !!profile.name ? profile.name : (doc?.name ?? "—");
```

Comment:
Logical OR should work here to make it a bit cleaner:

```suggestion
      options[profile._id] = profile.name || doc?.name ?? "—";
```

Reply:
I think you can't actually use `||` followed by `??` like that, it will throw a syntax error.